Makefile

```diff
@@ -178,11 +178,32 @@ distribution:
 # Build lib and frontend, and then run 'distribution'.
 package: mini-devel frontend install distribution
 
+.PHONY: conda-distribution
+# Create conda distribution files in lib/conda-recipe/dist.
+conda-distribution:
+	rm -rf lib/conda-recipe/dist
+	mkdir lib/conda-recipe/dist
+	# This can take upwards of 20 minutes to complete in a fresh conda installation! (Dependency solving is slow.)
+	GIT_HASH=$$(git rev-parse --short HEAD) conda build lib/conda-recipe --output-folder lib/conda-recipe/dist
```

Comment:
Should we add an inline comment telling people they need to have `conda` and `conda-build` installed? (We don't currently call these requirements out anywhere else, I think)

Reply:
Done! I'll also make some edits to the contributing wiki in case someone other than us wants to create conda builds on their local machine